P/PETSc/build_tarballs.jl

```diff
@@ -1,4 +1,7 @@
 using BinaryBuilder
+using Base.BinaryPlatforms
+const YGGDRASIL_DIR = "../.."
+include(joinpath(YGGDRASIL_DIR, "platforms", "mpi.jl"))
 
 name = "PETSc"
 version = v"3.16.5"
```

Comment:
It's typically preferable adding new dependencies when changing version number.  There is a new upstream version now: 3.16.6

Reply:
Done.